lib/sbom/scm/hex/scm.ex

```diff
@@ -216,6 +217,38 @@ defmodule SBoM.SCM.Hex.SCM do
     version
   end
 
+  @doc """
+  Enhances dependency metadata by fetching additional information from the Hex API.
+
+  Returns a map with metadata fields (licenses, links, source_url) that can be
+  merged with existing dependency information. Returns an empty map on error.
+  """
+  @impl SCM
+  def enhance_metadata(_app, %{mix_config: _config}) do
```

Comment:
Having a mix config does not mean that we have the package installed locally / were able to get the metadata from the `mix.exs`.

We need to check if links etc. are present and only do it if not.